ros/lib/logger.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import os
+import sys
+import socket
+from threading import local
+from ros.lib.config import LOG_LEVEL
+from logstash_formatter import LogstashFormatterV1
+
+
+threadctx = local()
+
+
+def cloudwatch_config_values():
```

Comment:
If I understand correctly, this is initial version and it includes local setup details & not deployment change, does it make sense to include cloudwatch configs and clowder handling in this PR?